### PR TITLE
Replace showAlert() with loclaized one.

### PR DESCRIPTION
**Pull request by inflation**
- [x] This change has been discussed with the author.
- [x] It implements / fixes issue #295 .

---

**Description:**
Rewrite of #301.

### CODE REVIEW

File: iina/Utility.swift [hunk 1]
```diff
@@ -29,6 +30,34 @@ class Utility {
     alert.alertStyle = alertStyle
     alert.runModal()
   }
+  
+  static func showAlert(_ key: String, comment: String? = nil, arguments: [CVarArg]? = nil, alertStyle: NSAlertStyle = .critical) {
```
Comment (lhc70000) [inline iina/Utility.swift]: `alertStyle` -> `style`

**Comment by saagarjha** [comment]
You should probably squash these two commits together.

**Comment by alejx** [comment]
@saagarjha Can we squash commits before merging? I see the squash and merge option in the merge options.

**Comment by saagarjha** [comment]
Yes, you can using `git rebase`. The squash and merge will create 1 commit of the pull request, squashing and merging will create 2 commits, and just merging will create 3.

**Pull request merged**